docs/ops/arithmetic/Tan_1.md

```diff
@@ -6,32 +6,29 @@
 
 **Short description**: *Tan* performs element-wise tangent operation with given tensor.
 
-**Attributes**:
+**Detailed description**:  Operation takes one input tensor and performs the element-wise tangent function on a given input tensor, based on the following mathematical formula:
```

Comment:
please clarify `X` in radians or degrees.
What value will be returned in case inf floating values?

Reply:
@rkazants  could you help to review, thanks